ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentSideBar.tsx

```diff
@@ -34,15 +63,15 @@ function DeploymentSideBar() {
                     <FlexItem>
                         <TextContent>
                             <Text component={TextVariants.h1} className="pf-u-font-size-xl">
-                                visa-processor
+                                {deployment?.name}
```

Comment:
shouldn't we have this all the time via the model?

Reply:
The `deployment` value actually comes from the API to get the deployment details so that's why it could be undefined until the API returns a response. You're right though, we could find the deployment name through the selected node in the model, but I thought it made sense to use the `deployment` value. We can get around this by having a loading state and empty state

Reply:
oh i see. yeah i think it makes sense to use both the graph deployment data and the deployment API response since we're using both for the side panel (at least for the flows tab) anyways